From 0a471eff34696b8f1193574c205c74416618bbbf Mon Sep 17 00:00:00 2001
From: Dan Voiculeasa <dan.voiculeasa@windriver.com>
Date: Thu, 30 Sep 2021 17:50:58 +0300
Subject: [PATCH 2/2] Add drbd.service file

Copy file from CentOS.
Use service from /lib/drbd instead of /etc/rc.d/init.d/.

Signed-off-by: Dan Voiculeasa <dan.voiculeasa@windriver.com>
---
 debian/drbd.service | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)
 create mode 100644 debian/drbd.service

diff --git a/debian/drbd.service b/debian/drbd.service
new file mode 100644
index 0000000..7624a18
--- /dev/null
+++ b/debian/drbd.service
@@ -0,0 +1,17 @@
+[Unit]
+Description=Control DRBD resources.
+After=network.target sshd.service
+
+[Service]
+Type=forking
+Restart=no
+KillMode=process
+RemainAfterExit=yes
+ExecStart=/lib/drbd/drbd start
+ExecStop=/lib/drbd//drbd stop
+ExecReload=/lib/drbd/drbd reload
+TimeoutSec=5min
+
+[Install]
+WantedBy=multi-user.target
+
-- 
2.30.0

